### PR TITLE
[MINOR] Fix for Member List

### PR DESCRIPTION
**Pull request by AHIRAISSHY**
# Changelogs

- Fixes the member list css colors
- Fixes the part where the last offline member was half shown

Fix1 FixGC
mlFix mlFix2

### CODE REVIEW

**Comment by AHIRAISSHY** [comment]
file*

**Comment by AHIRAISSHY** [comment]
that was an accident

File: src/plugins/membercount.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* Member Count Plugin */
 .members-1998pB #MemberCount {
-    background: var(--nox-level2)!important;
```
Comment (zerebos) [inline src/plugins/membercount.scss:3]: Why move away from using variables?

Comment (AHIRAISSHY) [reply src/plugins/membercount.scss:3]: i'ma need to refix

**Pull request closed without merging**